From 20cc633b19bb5bc7083a46fa4fb0f60bd79cb06a Mon Sep 17 00:00:00 2001
From: Subbaraya Sundeep Bhatta <subbaraya.sundeep.bhatta@xilinx.com>
Date: Mon, 23 Sep 2013 21:20:48 +0530
Subject: [PATCH 558/628] usb: gadget: Move match table above the
 usb_of_driver

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit 439e697f6b46ce02b9583ee32f7f61d21af9ba24

Moved match table above usb_of_driver

Signed-off-by: Subbaraya Sundeep Bhatta <sbhatta@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 drivers/usb/gadget/xilinx_udc.c |   16 +++++++---------
 1 files changed, 7 insertions(+), 9 deletions(-)

diff --git a/drivers/usb/gadget/xilinx_udc.c b/drivers/usb/gadget/xilinx_udc.c
index aca89ca..77fd2e3 100644
--- a/drivers/usb/gadget/xilinx_udc.c
+++ b/drivers/usb/gadget/xilinx_udc.c
@@ -38,15 +38,6 @@
 #include <linux/dma-mapping.h>
 #include "gadget_chips.h"
 
-
- /* Match table for of_platform binding */
-static const struct of_device_id usb_of_match[] = {
-	{.compatible = "xlnx,xps-usb2-device-4.00.a",},
-	{ /* end of list */ },
-};
-
-MODULE_DEVICE_TABLE(of, usb_of_match);
-
 /****************************************************************************
 DEBUG utilities
 *****************************************************************************/
@@ -2265,6 +2256,13 @@ static int usb_of_remove(struct platform_device *pdev)
 	return xudc_remove(pdev);
 }
 
+/* Match table for of_platform binding */
+static const struct of_device_id usb_of_match[] = {
+	{ .compatible = "xlnx,xps-usb2-device-4.00.a", },
+	{ /* end of list */ },
+};
+MODULE_DEVICE_TABLE(of, usb_of_match);
+
 static struct platform_driver usb_of_driver = {
 	.driver = {
 		.name = driver_name,
-- 
1.7.5.4

